tensor2tensor/data_generators/gym_utils.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, env, warm_up_examples=0, warmup_action=0):
     self.warm_up_examples = warm_up_examples
     self.warm_up_action = warmup_action
     self.observation_space = gym.spaces.Box(
-        low=0, high=255, shape=(210, 160, 3), dtype=np.uint8)
+        low=0, high=255, shape=(210, 160, 3), dtype=np.int32)
```

Comment:
Do we need int32? If these are uint8, why not keep them like this? If they're not, please add a comment why.